roles/parallels/tasks/main.yml

```diff
@@ -30,13 +30,47 @@
 # 4C6364ACXT|com.parallels.kext.hypervisor|1|Parallels International GmbH|1
 # 4C6364ACXT|com.parallels.kext.usbconnect|1|Parallels International GmbH|1
 #
-- name: Check if Paralles kernel extensions are approved
+- name: Check if Parallels kernel extensions are approved
   command: sqlite3 /var/db/SystemPolicyConfiguration/KextPolicy 'SELECT COUNT(*) from kext_policy WHERE team_id = "4C6364ACXT";'
   register: kernel_extension_policy
   changed_when: no
   tags:
     - parallels
     - parallels_check
+  when:
+    - parallels_assert_kexts
```

Comment:
I had to look up the [nuance on bare vars for conditionals](https://docs.ansible.com/ansible/latest/porting_guides/porting_guide_2.8.html#bare-variables-in-conditionals). It seems like that's no longer a concern (and only mattered for strings that looked like bools anyway).